nodejs.org/s2i/bin/save-artifacts

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+pushd ${HOME} >/dev/null
```

Comment:
I have no idea, only doing the same as official example :]
https://github.com/openshift/source-to-image/blob/master/docs/builder_image.md#example-save-artifacts-script